.pipelines/windows-nvda.yml

```diff
@@ -136,23 +77,31 @@ steps:
     path: helper/aria-at
     displayName: Checkout aria-at
   - task: PowerShell@2
-    displayName: Checkout aria-at to ${{ parameters['aria_at_ref'] }}
+    displayName: Checkout aria-at to $(aria_at_ref)
     inputs:
       targetType: "inline"
       workingDirectory: $(helper_dir)/aria-at
-      script: git checkout ${{ parameters['aria_at_ref'] }}
+      script: git checkout ${env:aria_at_ref}
   - checkout: nvda-at-automation
     path: helper/nvda-at-automation
     displayName: Checkout nvda-at-automation
   - task: DownloadGitHubRelease@0
-    displayName: Download nvda portable ${{ parameters['nvda_version'] }}
+    condition: eq(variables.nvda_version, 'latest')
+    displayName: Download latest nvda portable
+    # https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/download-github-release-v0?view=azure-pipelines
+    inputs:
+      connection: MyGitHubServiceConnection
+      userRepository: bocoup/aria-at-automation-nvda-builds
+      downloadPath: $(helper_dir)/nvda-portable
+  - task: DownloadGitHubRelease@0
+    condition: ne(variables.nvda_version, 'latest')
+    displayName: Download nvda portable $(nvda_version)
     # https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/download-github-release-v0?view=azure-pipelines
     inputs:
       connection: MyGitHubServiceConnection
       userRepository: bocoup/aria-at-automation-nvda-builds
-      ${{ if ne(parameters['nvda_version'], 'latest')}}:
-        defaultVersionType: specificTag
-        version: ${{ parameters['nvda_version'] }}
```

Comment:
this kind of YAML conditional doesn't work with variables because they aren't available at "compile time" where this if statement would be evaluated